ArduCopter/Copter.h

```diff
@@ -686,7 +686,7 @@ class Copter : public AP_HAL::HAL::Callbacks {
     float get_look_ahead_yaw();
     void update_throttle_hover();
     void set_throttle_takeoff();
-    float get_pilot_desired_throttle(int16_t throttle_control);
+    float get_pilot_desired_throttle(int16_t throttle_control, float thr_mid);
```

Comment:
It looks like thr_mid is by default 0 (it is added in several places), so we could add it here as the default value.